pom.xml

```diff
@@ -41,116 +40,47 @@
-      </bundle.build.name>
-      <bundle.file.name>
-        ${bundle.build.name}/${project.build.finalName}.jar
-      </bundle.file.name>
+      <sling.java.version>8</sling.java.version>
+      <project.build.outputTimestamp>1</project.build.outputTimestamp>
+      <pax-exam.version>4.13.3</pax-exam.version>
     </properties>
 
   <build>
     <plugins>
       <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-bundle-plugin</artifactId>
-        <extensions>true</extensions>
-        <configuration>
-          <instructions>
-            <Embed-Dependency>
-              org.apache.sling.commons.osgi;inline=org/apache/sling/commons/osgi/PropertiesUtil.class
-            </Embed-Dependency>
-          </instructions>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.felix</groupId>
-        <artifactId>maven-scr-plugin</artifactId>
-        <version>1.26.4</version>    
-      </plugin>
-
-      <!-- Required for pax exam-->
-
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>build-helper-maven-plugin</artifactId>
-        <version>1.8</version>
-        <executions>
-          <execution>
-            <id>reserve-network-port</id>
-            <goals>
-              <goal>reserve-network-port</goal>
-            </goals>
-            <phase>pre-integration-test</phase>
-            <configuration>
-              <portNames>
-                <portName>http.port</portName>
-              </portNames>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>properties-maven-plugin</artifactId>
-        <version>1.0-alpha-2</version>
-        <executions>
-          <execution>
-            <goals>
-              <goal>set-system-properties</goal>
-            </goals>
-            <phase>pre-integration-test</phase>
-            <configuration>
-              <properties>
-                <property>
-                  <name>project.bundle.file</name>
-                  <value>${bundle.file.name}</value>
-                </property>
-                <property>
-                  <name>http.port</name>
-                  <value>${http.port}</value>
-                </property>
-              </properties>
-            </configuration>
-          </execution>
-        </executions>
+        <groupId>biz.aQute.bnd</groupId>
+        <artifactId>bnd-maven-plugin</artifactId>
       </plugin>
       <plugin>
         <groupId>org.apache.servicemix.tooling</groupId>
         <artifactId>depends-maven-plugin</artifactId>
-        <version>1.2</version>
-        <executions>
-          <execution>
-            <id>generate-depends-file</id>
-            <goals>
-              <goal>generate-depends-file</goal>
-            </goals>
-          </execution>
-        </executions>
       </plugin>
+      <plugin>
```

Comment:
Can this be removed? It should be inherited from the parent pom.

Reply:
My bad, yes just confirmed its inherited from parent ✅